.gitignore

```diff
@@ -1,3 +1,19 @@
 /target
 .py03-env
 /data
+*.swp
```

Comment:
What a story this file tells about developing this.

Reply:
guilty as charged